### PR TITLE
Creating configs Naming Package

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Part of #52

### CODE REVIEW

**Comment by romani** [comment]
@relentless-pursuit , please remove from all PRs update in MainsLauncherTest.java
to let me merge all of such updates

**Comment by romani** [comment]
@relentless-pursuit , there is conflict

**Comment by relentless-pursuit** [comment]
> @relentless-pursuit , there is conflict

done

File: AbbreviationAsWordInName/all-examples-in-one/config.xml [hunk 1]
```diff
@@ -1,89 +0,0 @@
-<?xml version="1.0"?>
-<!DOCTYPE module PUBLIC
-        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
```
Comment (romani) [inline AbbreviationAsWordInName/all-examples-in-one/config.xml:3]: File should be just config.xml do not change name.

**Review by romani — APPROVED** [review]

**Pull request merged**